frame/nomination-pools/benchmarking/src/lib.rs

```diff
@@ -133,13 +137,28 @@ impl<T: Config> ListScenario<T> {
 
 		// Create accounts with the origin weight
 		let (pool_creator1, pool_origin1) = create_pool_account::<T>(USER_SEED + 1, origin_weight);
+		Pools::<T>::set_commission(
+			RuntimeOrigin::Signed(pool_creator1.clone()).into(),
```

Comment:
Should pref. go inside of `create_pool_account` because now you are assuming this id 1 and 2, while if you do it inside the function, it will be a bit more cohesive, in either case if you want to know the id of the last created pool, it is stored in some storge (`LastPoolId`).

Reply:
Some `create_pool_account`s are used assuming no commission is set, so I think a good middle ground is to pass a commission into it as a 3rd argument.

Reply:
I've done the above, I hope this is idiomatic.